x-pack/plugins/security/index.js

```diff
@@ -123,9 +129,23 @@ export const security = (kibana) => new kibana.Plugin({
         mappings,
         onBeforeWrite
       }) => {
-        const hasPrivileges = hasPrivilegesWithRequest(request);
-
         const adminCluster = server.plugins.elasticsearch.getCluster('admin');
+
+        if (!xpackInfo.feature(pluginId).getLicenseCheckResults().allowRbac) {
+          const { callWithRequest } = server.plugins.elasticsearch.getCluster('admin');
```

Comment:
`adminCluster` is defined above, can we use it here instead of getting the cluster again?